### PR TITLE
Add new filters

### PR DESCRIPTION
**Pull request by g1nt0ki**

### CODE REVIEW

File: src/content_script.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(async () => {
+  const src = chrome.runtime.getURL("src/pages/content/index.js")
```
Comment (0xngmi) [inline src/content_script.js:2]: is this to get around the firefox limitation around imports?

File: src/pages/background/index.ts [hunk 2]
```diff
@@ -207,37 +213,36 @@ function setupUpdateDomainDbs() {
...
       updateDomainDbs();
-      Browser.alarms.create("updateDomainDbs", { periodInMinutes: 1 }); // update once every 2 hours
+      Browser.alarms.create("updateDomainDbs", { periodInMinutes: 4 * 60  }); // update once every 4 hours
     }
   });
 }
 
-
-function setupUpdateTwitterConfig() {
+/* function setupUpdateTwitterConfig() {
```
Comment (0xngmi) [inline src/pages/background/index.ts:221]: imo better to just completely remove this rather than comment it out

**Review by 0xngmi — APPROVED** [review]

File: src/content_script.js (hunk 1 above)
Comment (g1nt0ki) [reply src/content_script.js:2]: for some reason, I ran into the same issue for chrome today

File: src/pages/background/index.ts (hunk 2 above)
Comment (g1nt0ki) [reply src/pages/background/index.ts:221]: removed it

**Pull request merged**